plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/ProjectSpecRunner.kt

```diff
@@ -195,21 +194,20 @@ private fun writeSettingFile(path: Path) {
 private fun ProjectSpec.writeGradleProperties(path: Path, options: TestOptions) {
   path.writeText(
     buildString {
-      appendLine("org.gradle.vfs.watch=false")
```

Comment:
This was intentional because we don't need file system watching when running Gradle through test kit.